python/python/glide/sync/glide_client.py

```diff
@@ -119,68 +162,102 @@ def _init_ffi(self):
             void free_command_response(CommandResponse* command_response_ptr);
             void free_error_message(char* error_message);
             void free_command_result(CommandResult* command_result_ptr);
-            CommandResult* command(const void* client_adapter_ptr, uintptr_t channel, int command_type, unsigned long arg_count, const size_t *args, const unsigned long* args_len, const unsigned char* route_bytes, size_t route_bytes_len);
+            CommandResult* command(
+                const void* client_adapter_ptr, uintptr_t channel, int command_type,
+                unsigned long arg_count, const size_t *args, const unsigned long* args_len,
+                const unsigned char* route_bytes, size_t route_bytes_len
+            );
 
-        """)
+        """
+        )
 
         # Load the shared library (adjust the path to your compiled Rust library)
-        self.lib = self.ffi.dlopen("/home/ubuntu/glide-for-redis/go/target/debug/libglide_rs.so")
-        
+        this_dir = os.path.dirname(__file__)
+        so_path = os.path.abspath(
+            os.path.join(this_dir, "../../../../ffi/target/debug/libglide_ffi.so")
+        )
+        self.lib = self.ffi.dlopen(so_path)
```

Comment:
Lets refactor it a bit, and have some consts up in the file:
```
from pathlib import Path

CURR_DIR = Path(__file__).resolve().parent
ROOT_DIR = CURR_DIR.parent.parent.parent.parent
FFI_DIR = ROOT_DIR / "ffi"
SO_FILE = FFI_DIR / "target" / "debug" / "libglide_ffi.so"
```
then we can do here:
```
self.lib = self.ffi.dlopen(str(SO_FILE.resolve()))